pkg/cluster/fetch.go

```diff
@@ -71,12 +68,8 @@ func GetEksdReleaseForCluster(ctx context.Context, cluster *v1alpha1.Cluster, bu
 	}
 	eksd, err := fetch(ctx, versionsBundle.EksD.Name, constants.EksaSystemNamespace)
 	if err != nil {
-		if strings.Contains(err.Error(), eksdReleaseObjDoesNotExist) {
-			logger.V(4).Info("EKS-D release objects are not present on the cluster. EKS-D release manifest will be retrieved from the URL in the bundle")
-			return nil, nil
-		} else {
-			return nil, fmt.Errorf("failed fetching EKS-D release for cluster: %v", err)
-		}
+		logger.V(4).Info("EKS-D release objects could not be fetched from the cluster. EKS-D release manifest will be retrieved from the URL in the bundle")
+		return nil, nil
```

Comment:
Instead of trying to string match all of the potential errors from kubectl when the release objs aren't there, we will just default to the normal behavior of getting it from the URL from the versions bundle if the EKS-D release objects cannot be fetched from the cluster